src/common/utils/validator.js

```diff
@@ -327,20 +342,11 @@ function isValidEmailLength(value) {
   return value.length <= MAX_EMAIL_LENGTH;
 }
 
-/**
- * Check if string length is within the limit
- * @param {String} value
- * @param {Bool}
- */
-function isValidTextBoxLength(value) {
-  return value.length <= MAX_TEXT_BOX_LENGTH;
-}
-
 /**
  * Parse datetime string to moment object
  * @param {String} dateTimeStr 'yyyy-MM-dd HH:mm:ss'
  * @param {String} dateTimeFormat defines the moment format default: 'YYYY-MM-DD HH:mm:ss'
- * @return {Bool}
+ * @return {Bool}ß
```

Comment:
?

Reply:
oops, result of using a different keyboard.